contrib/release.sh

```diff
@@ -88,6 +88,14 @@ else
    ./contrib/build-linux/sdist/build.sh
 fi
 
+# create source tarball
+srctarball="Electrum-source-$VERSION.tar.gz"
```

Comment:
Please see how `$tarball` is used in the rest of the file, and add the analogous code.
In particular, the non-releasemanager codepath needs to compare that they built the same distributable as the releasemanager.

Reply:
Adding the analogous code that applies for `$tarball` as well as for `$appimage`.